docs/qcqmc/index.md

```diff
@@ -0,0 +1,26 @@
+# QCQMC
+
+Notebooks outlining how to reproduce the results of [Unbiasing fermionic quantum Monte Carlo with a quantum computer](https://www.nature.com/articles/s41586-021-04351-z).
+Quantum Monte Carlo methods are a class of classical algorithms that can offer
+an efficient solution to the many-electron schroedinger equation but are plagued
+by the `fermion sign problem`. Typically a constraint is introduced to overcome
```

Comment:
nit: I think there should be a comma after Typically.

Reply:
Done.